iidm/iidm-converter-api/src/main/java/com/powsybl/iidm/IidmImportExportMode.java

```diff
@@ -8,12 +8,14 @@
 
 /***
  * @author Chamseddine BENHAMED <chamseddine.benhamed at rte-france.com>
+ *
+ * @deprecated Not used anymore
  */
-
+@Deprecated
 public enum IidmImportExportMode {
     ONE_SEPARATED_FILE_PER_EXTENSION_TYPE,
-    UNIQUE_FILE,
-    EXTENSIONS_IN_ONE_SEPARATED_FILE
+    EXTENSIONS_IN_ONE_SEPARATED_FILE,
+    UNIQUE_FILE
```

Comment:
You should revert this change: if bad people use the order of this enum, they can consider it as a regression. Keep only the `@Deprecated`